src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+# Python script to render 1D data written using SpECTRE in format v2
+
+import glob
+import h5py
+import argparse
+import sys
+import os
+import numpy as np
+import matplotlib as mpl
+mpl.use('Agg')
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import tkinter
+
+
+def find_extrema_over_data_set(arr):
+    '''
+    Find max and min over a range of number arrays
+    :param arr: the array over which to find the max and min
+    This doesn't really work if there are nan valued data pts.
+    '''
+
+    return np.nanmin(np.asarray(arr)), np.nanmax(np.asarray(arr))
+
+
+def parse_cmd_line():
+    '''
+    parse command-line arguments
+    :return: dictionary of the command-line args, dashes are underscores
+    '''
+
+    parser = argparse.ArgumentParser(\
+                                     description='Render 1-dimensional data',
+                                     formatter_class=
+                                     argparse.ArgumentDefaultsHelpFormatter)
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('--file-prefix',
+                       type=str,
+                       help='Common prefix of all h5 files being used '
+                       'in the case of a single h5 file, it is the file name '
+                       'without the .h5 extension')
+    group.add_argument('--filename-list',
```

Comment:
Similar to `--file-prefix`, on bad input this prints "Could not find file BurgersVolume0.h5zzz in directory" without telling me what directory.